backend/graminsta/core/models.py

```diff
@@ -12,5 +12,9 @@ class UserInfo(models.Model):
     """UserInfo Class
     """
     user = models.OneToOneField(auth_models.User, on_delete=models.CASCADE)
-    age = models.IntegerField()
-    gender = models.CharField(max_length=10)
+    age = models.PositiveIntegerField()
+
+    GENDER_MALE = 0
+    GENDER_FEMALE = 1
+    GENDER_CHOICES = [(GENDER_MALE, 'Male'), (GENDER_FEMALE, 'Female')]
```

Comment:
Move the definitions of the genders above, before all the `models.xxx` declarations. 

Nit: use Tuple (`GENDER_CHOICES = ((GENDER_MALE, ...))`) for `GENDER_CHOICES`.